cves/kernel/CVE-2021-3640.yml

```diff
@@ -232,7 +237,11 @@ subsystem:
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
   name:
-  note:
+  - bluetooth
+  - hci
+  note: |
+    path of the source code file is /net/bluetooth/sco.c. Also listed in
+    description as HCI subsystem
 interesting_commits:
```

Comment:
If there is no interesting commits, then there should be an explanation of why. Rudimentary fixing? Single commit fix?